Cargo.toml

```diff
@@ -42,3 +42,8 @@ members = [
 # Shutdown when panicking so we can see the error, specifically for the wallet
 [profile.release]
 panic = 'abort'
+
+[patch.crates-io]
+# Temporarily lock pgp to commit (master branch at time of writing) because the currently release crate locks zeroize to =1.3
+#pgp = { git = "https://github.com/rpgp/rpgp.git", rev = "21081b6aaaaa5750ab937cfef30bae879a740d23" }
```

Comment:
We can use this if https://github.com/tari-project/bulletproofs/pull/13 is tagged and published